bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestClusterAPI.java

```diff
@@ -157,7 +164,11 @@ private void waitForOperation(BigtableClusterAdminClient client, String operatio
         e.printStackTrace();
```

Comment:
Also, since you are modifying the logic, We should wait after doing a first check, and correctly propagate the InterruptedException.